packages/sanity/src/desk/i18n/resources.ts

```diff
@@ -165,6 +165,136 @@ const deskLocaleStrings = {
 
   /** --- "PRODUCTION PREVIEW", eg link to content --- */
   'production-preview.menu-item.title': 'Open preview',
+
+  /** -- DESK PANES -- */
+  /** The tool tipe for the split pane button on the document panel header */
+  'document-panel-header.split-pane-button.tooltip': 'Split pane right',
+
+  /** The text content for the deleted document banner */
+  'banners.deleted-document-banner.text': 'This document has been deleted.',
+
+  /** The text for the restore button on the deleted document banner */
+  'banners.deleted-document-banner.restore-button.text': 'Restore most recent version',
+
+  /** The text for the reference change banner if the reason is that the reference has been changed */
+  'banners.reference-changed-banner.reason-changed.text':
+    'This reference has changed since you opened it.',
+
+  /** The text for the reload button */
+  'banners.reference-changed-banner.reason-changed.reload-button.text': 'Reload reference',
+
+  /** The text for the reference change banner if the reason is that the reference has been deleted */
+  'banners.reference-changed-banner.reason-removed.text':
+    'This reference has been removed since you opened it.',
+
+  /** The text for the close button */
+  'banners.reference-changed-banner.reason-removed.close-button.text': 'Close reference',
+
+  /** The text for the permission check banner if there is only one role */
+  'banners.permission-check-banner.singular-role.text':
+    'Your role {{roles}} does not have permissions to {{requiredPermission}} this document.',
+
+  /** The text for the permission check banner if there is are multiple roles */
+  'banners.permission-check-banner.plural-roles.text':
+    'Your roles {{roles}} do not have permissions to {{requiredPermission}} this document.',
+
+  /** The text for when a form is hidden */
+  'document-view.form-view.form-hidden': 'This form is hidden',
+
+  /** The text for when the form view is loading a document */
+  'document-view.form-view.loading': 'Loading document',
+
+  /** The title of the sync lock toast on the form view */
+  'document-view.form-view.sync-lock-toast.title': 'Syncing document…',
+
+  /** The description of the sync lock toast on the form view */
+  'document-view.form-view.sync-lock-toast.description':
+    'Please hold tight while the document is synced. This usually happens right after the document has been published, and it should not take more than a few seconds',
+
+  /** The title of the reconnecting toast */
+  'panes.document-pane-provider.reconnecting-toast.title': 'Connection lost. Reconnecting…',
```

Comment:
Similar here - ideally we would avoid "toast" as the UI element for displaying this message may change.